### DISCUSSION TITLE
Installed VDD beta Beta Virtual Display Driver (24.10.27), but no HDR support?

### DISCUSSION THREAD

**Discussion by Kazzahdrane** [VirtualDrivers/Virtual-Display-Driver#215] [Q&A]
![image](https://github.com/user-attachments/assets/bff0d728-a3b7-4da5-9637-e3de20b899bd)

Hi! I'd tried to use the first stable release with HDR many months ago, but couldn't get my PC to accept the driver at all. With the latest beta release, I'm delighted to say that I now see my VDD showing in Display Settings!

However, as you can see in the screenshot, Windows believes the virtual display doesn't support HDR. Do I need to make any changes to the vdd_settings.xml to enable this? My primary physical monitor supports HDR and I've confirmed it says as such in the display settings screen.

**Comment by itsmikethetech** [comment]
What specific OS version?

**Reply by Kazzahdrane** [reply]
Hi, I'm on:

Windows 11 Pro
10.0.22631 Build 22631

**Reply by bud3699** [reply]
Sun valley, do you have an update available in windows ???
